src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -896,7 +896,8 @@ const Authenticate = (() => {
                 service      : 'onfido',
             }).then((response) => {
                 if (response.error) {
-                    resolve();
+                    resolve(response.error.code);
+                    return;
```

Comment:
why do we need another return? doesn't resolve already return?

Reply:
rejecting or resolving won't stop the execution of the rest of the function. The function may contain code that will create confusing results

Reply:
I see, I always thought it did :D